src/main/java/org/opensearch/geospatial/search/aggregations/bucket/geogrid/GeoHexHelper.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.geospatial.search.aggregations.bucket.geogrid;
+
+import static org.opensearch.geospatial.h3.H3.h3IsValid;
+import static org.opensearch.geospatial.h3.H3.h3ToLatLng;
+import static org.opensearch.geospatial.h3.H3.stringToH3;
+
+import java.util.Locale;
+
+import lombok.NonNull;
+
+import org.opensearch.common.geo.GeoPoint;
+import org.opensearch.geospatial.h3.H3;
+
+public class GeoHexHelper {
+
+    public static int MIN_H3_RES = 0;
```

Comment:
should we move this to H3 class, as we have MAX_H3_RES defined there only.

Reply:
Was reluctant to touch the library since it was imported, but there is no reason not to add. I updated now.